CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 * feat(bindings/C): implement capability by @Ji-Xinyou in https://github.com/apache/incubator-opendal/pull/3479
 * feat: add mongodb gridfs service support by @realtaobo in https://github.com/apache/incubator-opendal/pull/3491
 * feat(services): add RedisConfig by @hoslo in https://github.com/apache/incubator-opendal/pull/3498
+* chore(service/vercel_artifacts): add doc in backend by @caicancai in https://github.com/apache/incubator-opendal/pull/3508
```

Comment:
@Xuanwo Is this in the wrong place? It shouldn't be in feat

Reply:
It seems github-action-bot adds 2 labels to PR #3508. We should remove one.

Reply:
Thank you for your reply